### ISSUE TITLE
Query undefined retrieve last entity

### ISSUE THREAD

**Issue by AlexandreBonaventure** [vuex-orm/vuex-orm#126]
version: 0.19.0

**Repro**
```js
getters['entities/{entityName}/query']().find(undefined)
=> EntityName { id: "b9a818f0-d700-11e7-a0b8-ed4d4ad02ecc" }

getters['entities/{entityName}/query']().find(null)
=> null
```
**What is expected ?**
Passing undefined to find method should not retrieve data. (Or maybe documentation should make this more explicit)

**Comment by kiaking** [comment]
Hi thanks for the report! This expected now but rethinking about it, I think we should change the behavior to return null if the `find` method is called without argument (equals to passing undefined).

Currently `find` method act as exactly as same as `first` method so it fetches the very first record if there's no argument. But yeah with the name `find`, I think it shouldn't be used without the argument.

**Comment by kiaking** [comment]
This is fixed with Vuex ORM 0.24.3 🎉 
https://github.com/vuex-orm/vuex-orm/releases/tag/v0.24.3
